war/src/main/js/upgradeWizard.js

```diff
@@ -1,40 +1,38 @@
-onSetupWizardInitialized(function(wizard) {
```

Comment:
I did that in other cases, the question is: Why do you need `onSetupWizardInitialized`?